### PR TITLE
Fix arguments xml parser

### PR DESCRIPTION
**Pull request by pedrovelho**
Add <arguments>  inside <script>, must be declared after <code>.

### CODE REVIEW

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [StaxJobFactory.java#L733](https://github.com/ow2-proactive/scheduling/blob/2f2915e5dfde895db08ef445fa6ebd863383ea12/scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/StaxJobFactory.java#L733): Add a default case to this switch. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
1. ![MINOR][MINOR] [Job2XMLTransformer.java#L78](https://github.com/ow2-proactive/scheduling/blob/2f2915e5dfde895db08ef445fa6ebd863383ea12/scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/Job2XMLTransformer.java#L78): Remove this unused import 'org.w3c.dom.CDATASection'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [StaxJobFactory.java#L733](https://github.com/ow2-proactive/scheduling/blob/9d442e6621e445198f05909fa5530f0551b09b9d/scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/StaxJobFactory.java#L733): Add a default case to this switch. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: scheduler/scheduler-api/src/test/java/org/ow2/proactive/scheduler/common/job/factories/Job2XMLTransformerTest.java [hunk 1]
```diff
@@ -86,6 +87,24 @@ public void forkEnvironmentIsPreserved() throws Exception {
...
+        SimpleScript script = new SimpleScript("\nprint('arguments[0]='+arguments[0])\n", "javascript", params);
+        ScriptTask task = new ScriptTask();
+        task.setName("testTask");
+        task.setScript(new TaskScript(script));
+        job.addTask(task);
+
+        new Job2XMLTransformer().job2xmlFile(job, xmlFile);
+        TaskFlowJob recreatedJob = (TaskFlowJob) (JobFactory.getFactory().createJob(xmlFile.getAbsolutePath()));
+
+        assertNotNull(((ScriptTask) recreatedJob.getTask("testTask")).getScript());
```
Comment (fviale) [inline scheduler/scheduler-api/src/test/java/org/ow2/proactive/scheduler/common/job/factories/Job2XMLTransformerTest.java]: you need to test the parameters

File: scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/StaxJobFactory.java [hunk 2]
```diff
@@ -1285,6 +1292,10 @@ else if (XMLTags.TOPOLOGY_DIFFERENT_HOSTS_EXCLUSIVE.matches(cursorTask.getLocalN
                                 attrtmp = null;
                                 throw new JobCreationException("Invalid script file definition, one of path/url attributes must be declared");
                             }
+                        } else if (XMLTags.SCRIPT_ARGUMENTS.matches(current)) {
+                            toReturn = new SimpleScript(toReturn.getScript(),
```
Comment (fviale) [inline scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/StaxJobFactory.java]: alternatively, you could add a setParameters method in Script class.

File: scheduler/scheduler-api/src/test/java/org/ow2/proactive/scheduler/common/job/factories/Job2XMLTransformerTest.java (hunk 1 above)
Comment (pedrovelho) [reply scheduler/scheduler-api/src/test/java/org/ow2/proactive/scheduler/common/job/factories/Job2XMLTransformerTest.java]: Lol, indeed my bad.

**Pull request merged**